src/deploy.py

```diff
@@ -1,48 +1,15 @@
 """module concerns itself with tasks involving branch deployments of projects."""
 
 from fabric.api import task
-from decorators import requires_branch_deployable_project, echo_output, setdefault, requires_aws_stack, timeit
-import utils
-from buildercore import core, bootstrap, cfngen, project
+from decorators import requires_aws_stack
+from buildercore import bootstrap
 from buildercore.concurrency import concurrency_for
-import cfn
 import buildvars
 
 import logging
 
 LOG = logging.getLogger(__name__)
 
-def impose_ordering(branch_list):
-    branch_list, removed = utils.rmval(branch_list, 'master', 'develop')
-    branch_list.sort()
-    branch_list = removed + branch_list
-    return branch_list
-
-#
-#
-#
-
-@task
-@requires_branch_deployable_project
-@echo_output
-@timeit
-def deploy(pname, instance_id=None, alt_config=None):
```

Comment:
it seems this has been completely absorbed by `launch` now

Reply:
my thoughts too, and they were both using `bootstrap.create_update` and `create_update` was only ever a convenience function, so I absorbed that into `cfn.launch`.

`deploy.switch_revision_update_instance` should probably be moved to `cfn.py`, but that can happen later. I feel there is a refactor/clean up in builder's immediate future with the move to python 3